src/applications/vaos/services/appointment/transformers.v2.js

```diff
@@ -212,8 +212,8 @@ export function transformVAOSAppointment(appt) {
                 firstName: provider.name?.given,
                 lastName: provider.name?.family,
               },
-              providerName: `${provider.name?.given} ${provider.name?.family}`,
             })),
+            providerName: getProviderName(appt),
```

Comment:
@vbahinwillit Aah you got me! I would like to have each practitioner its own display name but I don't know how to iterate through a complex array object. So I opted to display the first practitioner name  :-) Any suggestion?

Reply:
@jenniemc - The providerName could be added to the loop and concatenated together like:

```
providers: (providers || []).map(provider => ({
              name: {
                firstName: provider.name?.given.join(), // I would also join this too since not joining assigns the array to the firstName. The default is to join separated by a comma
                lastName: provider.name?.family,
              },
              providerName: provider.name
                ? `${provider.name.given.join()} ${provider.name.family}`
                : null,
            }))
```

This should also be changed in ./services/appointments/index.js:getProviderName:932